src/V3LinkDot.cpp

```diff
@@ -1017,9 +1020,10 @@ class LinkDotFindVisitor final : public VNVisitor {
             for (AstNode* stmtp = nodep->stmtsp(); stmtp; stmtp = stmtp->nextp()) {
                 if (VN_IS(stmtp, Var) || VN_IS(stmtp, Foreach)) {
                     std::string name;
+                    const std::string stepStr = std::to_string(m_statep->stepNumber()) + "_";
```

Comment:
```suggestion
                    const std::string stepStr = m_statep->step() == PRIMARY ? "" : std::to_string(m_statep->stepNumber()) + "_";
```
This way most of the changes to the test golden results should go away.